Modules/Setup.dist

```diff
@@ -394,3 +397,11 @@ _symtable symtablemodule.c
 
 # Another example -- the 'xxsubtype' module shows C-level subtyping in action
 xxsubtype xxsubtype.c
+
+# Uncommenting the following line tells makesetup that all following modules
+# are not built (see above for more detail).
+#
+#*disabled*
+#
+#_sqlite3 _tkinter _curses pyexpat
```

Comment:
It makes me a little sad that this is supported syntax for the Setup file (multiple module names on a line), but since this is already the case, might as well use it.